### PR TITLE
adding description and url

### PR DESCRIPTION
**Pull request by rjeschmi**
Adding description and url to fpm commandline 

Will resolve: #1462

### CODE REVIEW

**Comment by boegel** [comment]
@rjeschmi: how about an (enhanced) unit test? see `test/framework/package.py`

**Comment by boegel** [comment]
Jenkins: ok to test

File: test/framework/package.py [hunk 1]
```diff
@@ -44,16 +44,70 @@
 
 
 MOCKED_FPM = """#!/bin/bash
+
+DEBUG=  #put something here if you want to debug 
+debug_echo () {
+    if [ -n "$DEBUG" ]; then
+        echo "$@" >> /tmp/debug_fpm_mock
```
Comment (boegel) [inline test/framework/package.py]: don't hardcode `/tmp/debug_fpm_mock`, use `$TMPDIR` which will be a unique directory when the script it run (you can echo where you're debugging to), or at the very least use `/tmp/$USER/debug_fpm_mock` (but I strongly prefer using `$TMPDIR`)

File: test/framework/package.py [hunk 2]
```diff
@@ -44,16 +44,70 @@
...
+            ;;
+        --*)
+            debug_echo "got a unhandled opption"
+            shift 2
+            ;;
+        *)
+            debug_echo "got the rest of the output"
+            installdir="$1"
+            modulefile="$2"
+            shift 2
```
Comment (boegel) [inline test/framework/package.py]: since you're always shifting 2, do this outside of the `case`?

File: test/framework/package.py [hunk 3]
```diff
@@ -44,16 +44,70 @@
...
+        "-t")
+            target="$2"
+            shift 2
+            ;;
+        "-s")
+            source="$2"
+            shift 2
+            ;;
+        --*)
+            debug_echo "got a unhandled opption"
```
Comment (boegel) [inline test/framework/package.py]: `option`

**Comment by rjeschmi** [comment]
Looks good. Tests all pass now anyway...

**Comment by boegel** [comment]
Thanks a lot @rjeschmi, great work on enhancing the tests!

**Pull request merged**